_posts/2015-12-13-impact.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Impact
+uuid: EF41545-82101-29BAD-E4114-sdd
+---
+
+I don't think I optimise my time for my happiness.
+
+There's a programmer whose impact is significantly bigger than mine, when I talk to them about this, they are not motivated at all by the scale of their day to day work in OSS. On the other hand, that motivates me.
+
+I love hanging out with the people at [Coalition for Queens](http://www.c4q.nyc/), they are people who have grit. They are willing to put their entire life on hold, each class of C4Q is 9 months long of weekends and almost daily after work hours. When you talk to them about that sacrifice, they talk about being able to make a bigger impact, on themselves, their family and their community.
+
+I started coming up with a way of debating what I should work on in the last few months,
+I call it impact per hour. I imagine it by impacting four separate places, Artsy, the Community, the World and Myself. Only one of these will actually optimise for my own happiness in the end. The others will probably increase general satisfaction, but that's not quite happiness.
+
+Yesterday, I heard someone say that they optimise for personal happiness, and I think that's cool. I'm not built that way; my work is draining; emotionally and physically.
+
+There's the obvious stuff in the time it takes to build things, the less obvious in the [long term ramifications](https://www.youtube.com/watch?v=e_-qV8waPVM) of OSS maintenance. Then there's the fact that some days you can just wake up to really de-motivating comments on the internet. It's important to remember you are not your products, though they are definitely a part of you.
```

Comment:
So I actually recognized this youtube video ID as @searls talk – it broke my regex for youtube URLs earlier on this: https://github.com/ashfurrow/blog/pull/177/files